tools/generator/game.py

```diff
@@ -165,6 +165,18 @@ def check_field_unicity(self):
                         .format(origin_type, origin, name, def_type, def_name))
                 used_field_names[name] = (origin_type, origin)
 
+    def get_struct(self, struct_name):
+        '''Get a struct by name, None if it does not exist'''
+        return next(
+            (
```

Comment:
I think this pair of parens is useless

Reply:
You can only omit them for single-argument function calls:

```python3
>>> next(i for i in range(10, 20), None)
  File "<stdin>", line 1
SyntaxError: Generator expression must be parenthesized if not sole argument
```